src/main/jbake/content/blog/2017/01/13/events-aggregation-extension.adoc

```diff
@@ -0,0 +1,189 @@
+= Extending Complex Event Processing in Hawkular Alerting
+Lucas Ponce
+2017-01-13
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerts, complex, events, processing, stream, polling
+
+== Stream Processing versus Polling Processing
+
+Hawkular Alerting uses different techniques to detect behaviours by defining rules.
+
+The main implementation is based in a Stream Processing design.
+Hawkular Alerting analyzes streams of data and events ordered in a timeline searching for matches against conditions.
+This technique is also known as Event Processing (or Complex Event Processing) specially when the processing of incoming data might need multiple conditions.
```

Comment:
I agree that Event Processing is sensitive to the context and the domain which is applied.
Even in the same context/domain as you point, there is not a ubiquitous definition with the term.
There are some common patterns like timeline streams of information processed, and the Complex prefix can be related to several scenarios, complexity of the derived inference (in the way of high level events or the rules that are defined to express these events).

So, I see your point. But I think in the scope of this post, and its domain (Hawkular Alerting) is valid to use these terms without enter in too much ambiguity.

Thanks for the note.